routes/api/children.js

```diff
@@ -0,0 +1,10 @@
+const childrenController = require('../../controllers/childrenController');
+
+module.exports = (app) =>{
+    app.post('/api/createChild',childrenController.createChild);
+    app.get("/api/findAllByChild", childrenController.findAllByChild);
+    app.get("api/findbyid", childrenController.findById);
+    app.put("/api/updatechildbudget", childrenController.updateChildBudget);
```

Comment:
seeing a mix of camelCase and lowercase in here. Not a big issue but always good to keep consistency.